tests/test_generate_diff.py

```diff
@@ -31,9 +31,9 @@ def test_generate_diff_yaml() -> None:
 
 def test_generate_diff_mixed() -> None:
     # given
+    file1 = str(resolve_path("fixtures/file_1" + ".yaml"))
+    file2 = str(resolve_path("fixtures/file_2" + ".json"))
```

Comment:
also, as you have two parts now (building diff dict and message formatting, you could split tests to have one for diff building and one for message formatting)